Sources/BlogModuleApi/Objects/BlogAuthor/BlogAuthorGetObject.swift

```diff
@@ -13,19 +13,26 @@ public struct BlogAuthorGetObject: Codable {
     public var name: String
     public var imageKey: String?
     public var bio: String?
-    public var links: [BlogAuthorLinkListObject]
+    public var updated_at: Date?
+    public var created_at: Date?
+    
+    public var links: [BlogAuthorLinkListObject]?
     
     public init(id: UUID,
                 name: String,
                 imageKey: String?,
                 bio: String?,
+                updated_at: Date?,
+                created_at: Date?,
                 links: [BlogAuthorLinkListObject])
     {
         self.id = id
         self.name = name
         self.imageKey = imageKey
         self.bio = bio
-        self.links = links
+        self.updated_at = updated_at
+        self.created_at = created_at
+        self.links = links.count > 0 ? links : nil
```

Comment:
@tib Here I have a problem with Eager load on the links if you try to retrieve them as I did in the Posts. 
So I actually, prefer not to return anything rather than an empty array.

Reply:
You can access the relations through the property wrapper with the $ syntax. e.g.: `$links.value` 😉